### PR TITLE
SONAR-9736 Build UI for long-living branches

### PR DESCRIPTION
**Pull request by stas-vilchik**

### CODE REVIEW

File: server/sonar-web/src/main/js/apps/overview/events/AnalysesList.js [hunk 1]
```diff
@@ -24,12 +24,14 @@ import Analysis from './Analysis';
...
 import { getMetrics } from '../../../api/metrics';
 import { getProjectActivity } from '../../../api/projectActivity';
+import { getBranchName } from '../../../helpers/branches';
 import { translate } from '../../../helpers/l10n';
 /*:: import type { Analysis as AnalysisType } from '../../projectActivity/types'; */
 /*:: import type { History, Metric } from '../types'; */
 
 /*::
 type Props = {
+  branch: {},
```
Comment (gregaubert) [inline server/sonar-web/src/main/js/apps/overview/events/AnalysesList.js]: The branch parameters should also be passed down to `PreviewGraph` component, currently clicking on it while on a long lived branch display the activity graph for the main branch.

**Comment by gregaubert** [comment]
This is probably not related to your PR but when I'm on a long lived branch and I click on an issue to see it in a file, the file don't load, I get an error message like this from `api/sources/lines` : `
{msg: "Component 'lodash:debounce.js:BRANCH:branch-1.0' on branch 'branch-1.0' not found"}`

File: server/sonar-web/src/main/js/apps/component-measures/drilldown/BubbleChart.js [hunk 2]
```diff
@@ -37,6 +37,7 @@ import { RATING_COLORS } from '../../../helpers/constants';
 const HEIGHT = 500;
 
 /*:: type Props = {|
+  branch: {},
```
Comment (gregaubert) [inline server/sonar-web/src/main/js/apps/component-measures/drilldown/BubbleChart.js]: Is it to force re-render the component when the branch changes ?

**Comment by stas-vilchik** [comment]
@gregaubert yeah, I just need to rebase my branch. Julien should have fixed the keys there.

File: server/sonar-web/src/main/js/apps/component-measures/drilldown/BubbleChart.js (hunk 2 above)
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/apps/component-measures/drilldown/BubbleChart.js]: not really... it's here by mistake

**Comment by stas-vilchik** [comment]
@gregaubert could you review 5 last commits please?

**Pull request merged**